tests/test_metrics_comparative.py

```diff
@@ -29,12 +29,10 @@ def test_binary_jaccard_reference(truth_binary_values, prediction_binary_values)
     cm = create_binary_confusion_matrix(truth_binary_values, prediction_binary_values)
 
     value = metrics.binary_jaccard(cm)
-    # sklearn has a very idiosyncratic implementation of jaccard_similarity_score; unless the input
-    # arrays are wrapped in an additional dimension, the result is actually the accuracy score
-    # see: https://github.com/scikit-learn/scikit-learn/issues/3037
-    reference_value = sklearn.metrics.jaccard_similarity_score(
+    reference_value = sklearn.metrics.jaccard_score(
         np.expand_dims(truth_binary_values, axis=0),
```

Comment:
Do we still need to wrap these with [`np.expand_dims`](https://docs.scipy.org/doc/numpy/reference/generated/numpy.expand_dims.html)? That was previously taking an array like `[1, 0, 1]` and wrapping it into `[[1, 0, 1]]`. Is the new implementation of `sklearn.metrics.jaccard_score` now able to take the arrays directly (like all the other `sklearn.metrics`)?